Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/AgendaDayViewModel.kt

```diff
@@ -0,0 +1,82 @@
+package com.mentalmachines.droidcon_boston.views.agenda
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Transformations
+import androidx.lifecycle.ViewModel
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseError
+import com.google.firebase.database.ValueEventListener
+import com.mentalmachines.droidcon_boston.data.FirebaseDatabase
+import com.mentalmachines.droidcon_boston.data.Schedule
+import com.mentalmachines.droidcon_boston.data.UserAgendaRepo
+import com.mentalmachines.droidcon_boston.firebase.FirebaseHelper
+import timber.log.Timber
+
+class AgendaDayViewModel(
+    private val dayFilter: String,
+    val onlyMyAgenda: Boolean,
+    private val userAgendaRepo: UserAgendaRepo
+) : ViewModel() {
+    private val firebaseHelper = FirebaseHelper.instance
+
+    private val _scheduleRows = MutableLiveData<List<Schedule.ScheduleRow>>()
```

Comment:
this is definitely a nit, but why have the `_` char before the private variable? I've seen this in Python, but not Kotlin yet

Reply:
So, the goal behind it is: Make the MutableLiveData private, so that it can't be manipulated by another class, and make the public version just a regular LiveData. The underscore convention was one I got from this article. Happy to change it if you think it's overkill: 

https://proandroiddev.com/5-common-mistakes-when-using-architecture-components-403e9899f4cb